plugin/core/registry.py

```diff
@@ -105,8 +104,19 @@ def sessions(self, capability: Optional[str] = None) -> Generator[Session, None,
         yield from sessions_for_view(self.view, capability)
 
 
-class LspRestartClientCommand(sublime_plugin.TextCommand):
+class LspRestartServerCommand(LspTextCommand):
     def run(self, edit: Any) -> None:
         window = self.view.window()
+        if not window:
+            return
+
+        sessions = [session.config.name for session in self.sessions()]
```

Comment:
Please call it something better. These are not sessions but config names rather.